### PR TITLE
SWG-9025 Fetch organization standardization configuration command

### PR DESCRIPTION
**Pull request by MiloszTarka**
This PR adds command that allows users to fetch their organization's standardization ruleset using Swaggerhub-CLI (Export functionality)

* [X] I have added the appropriate label to my PR

### CODE REVIEW

File: src/commands/api/validate/download-rules.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+class ValidateDownloadRulesCommand extends BaseCommand {
+    async run() {
+        const { args, flags } = await this.parse(ValidateDownloadRulesCommand)
+
+        const includeSystemRules = flags['include-system-rules'] ?? false
+        const includeDisabledRules = flags['include-disabled-rules'] ?? false
+        const organization = args['OWNER']
+
+        const organizationRuleset = await this.getExportedOrganizationRuleset(organization, {includeSystemRules, includeDisabledRules})
+        this.log(JSON.stringify(organizationRuleset, null, 2))
```
Comment (MateuszPol) [inline src/commands/api/validate/download-rules.js:16]: This logger is needed?

**Review by MateuszPol — APPROVED** [review]

File: src/commands/api/validate/download-rules.js (hunk 1 above)
Comment (MiloszTarka) [reply src/commands/api/validate/download-rules.js:16]: Its result printing command from oclif

Comment (MateuszPol) [reply src/commands/api/validate/download-rules.js:16]: Ok, great ;)

File: test/resources/rulesets.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+module.exports = {
+    ruleset,
+    rulesetWithSystemRule,
+    rulesetWithDisabledRule,
+    rulesetWithDisabledAndSystemRule
+}
+
+
+
+
```
Comment (CalemRoelofsSB) [inline test/resources/rulesets.js:68]: Don't need these 4 blank lines here ;)

**Review by CalemRoelofsSB — APPROVED** [review]
Absolutely perfect, good job man

File: test/resources/rulesets.js (hunk 2 above)
Comment (MiloszTarka) [reply test/resources/rulesets.js:68]: oh wow, thanks

**Pull request merged**